specs/binding-kafka.spec/src/main/scripts/io/aklivity/zilla/specs/binding/kafka/streams/application/fetch/isolation.read.uncommitted.aborted/client.rpt

```diff
@@ -90,6 +90,8 @@ read zilla:data.ext ${kafka:matchDataEx()
                              .build()}
 read "Hello, world"
 
+read notify SENT_ABORT
+
```

Comment:
Normally we think of barriers as events that need to be notified because they happened, rather than naming something yet to happen in the future.

So we should rename this to reflect what has already happened here.

Suggest something like `RECEIVED_SKIPPED_MESSAGE`?